### ISSUE TITLE
Fix the issue where json db does not store multi-indexed data frames properly

### ISSUE THREAD

**Issue by orbitfold** [UCL-CCS/EasyVVUQ#218]

**Comment by fjansson** [comment]
Here is an example dataframe, generated with the test_vector.py testcase which uses the json database. The problem is the "numbers" columns, which should be indexed with tuples.

```
Unnamed: 0      mu  sigma  num_steps    out_file    numbers  numbers.1   numbers.2  ...  numbers.4   numbers.5   numbers.6  numbers.7  numbers.8   numbers.9  run_id ensemble_id
0         NaN     NaN    NaN        NaN         NaN   0.000000   1.000000    2.000000  ...   4.000000    5.000000    6.000000   7.000000   8.000000    9.000000     NaN         NaN
1         0.0    1.00   0.25       10.0  output.csv   0.791982   0.803967    1.021388  ...   1.221006    0.883225    0.505340   1.029886   1.195682    1.311029   Run_1  Ensemble_1
2         1.0   25.75   0.25       10.0  output.csv  25.570410  25.739344   26.350087  ...  25.433332   25.843866   25.528776  25.492828  25.818198   25.652819   Run_2  Ensemble_2
3         2.0   75.25   0.25       10.0  output.csv  75.115109  75.443457   75.383208  ...  75.154449   75.285934   75.190648  75.615100  75.429117   75.787631   Run_3  Ensemble_3
4         3.0  100.00   0.25       10.0  output.csv  99.708222  99.755710  100.018658  ...  99.940213  100.191178  100.308861  99.860778  99.786449  100.308648   Run_4  Ensemble_4
```

When using the default sql database, the dataframe looks like this:
```
index  ('mu', '')  ('sigma', '')  ('num_steps', '') ('out_file', '')  ...  ('numbers', 7)  ('numbers', 8)  ('numbers', 9)  ('run_id', '')  ('ensemble_id', '')
0  None        1.00           0.25                 10       output.csv  ...        0.769484        1.309662        0.987631           Run_1           Ensemble_1
1  None       25.75           0.25                 10       output.csv  ...       26.020490       25.959411       25.563026           Run_2           Ensemble_2
2  None       75.25           0.25                 10       output.csv  ...       75.034389       75.520591       75.379132           Run_3           Ensemble_3
3  None      100.00           0.25                 10       output.csv  ...       99.864189      100.132343      100.304935           Run_4           Ensemble_4
```
...which is also wrong, because the indices in this case are strings: `"('numbers', 7)"`.

**Comment by orbitfold** [comment]
Solved with #220, will need to look in to the SQL case now.
